scripts/evals/run_rag_release_gates.py

```
@@ -0,0 +1,2172 @@
+def _write_flat_export(run_dir: Path, traces: Sequence[dict[str, Any]]) -> str:
+    """Write Parquet when available; otherwise fall back to CSV."""
+
+    flat_rows = _flat_trace_rows(traces)
+    csv_path = run_dir / "traces.csv"
+    parquet_path = run_dir / "traces.parquet"
+    try:
+        import pandas as pd
+
+        pd.DataFrame(flat_rows).to_parquet(parquet_path, index=False)
+        return str(parquet_path)
+    except Exception as exc:
```

Flake8 (7.3.0)

[error] local variable 'exc' is assigned to but never used

(F841)